README.md

```diff
@@ -1151,6 +1151,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
 ## IaaS
 
+  * [Cloudflare R2](https://www.cloudflare.com/products/r2/) - Free 10GB Object storage with S3-compatible API and Zero egress fee
```

Comment:
Better to add it to the existing cloudflare entry.  Thanks!

Reply:
The existing cloudflare entry is under CDN and ... 
I don't think it fits there
If I were to search for object storage, I won't be searching it in CDN

Reply:
At the top is a whole dedicate CF entry listing all their stuff. We try to list same company ilonly once.